stubs/opentracing/opentracing/mocktracer/context.pyi

```diff
@@ -6,4 +8,4 @@ class SpanContext(opentracing.SpanContext):
     def __init__(self, trace_id: int | None = ..., span_id: int | None = ..., baggage: dict[str, str] | None = ...) -> None: ...
     @property
     def baggage(self) -> dict[str, str]: ...
-    def with_baggage_item(self, key: str, value: str) -> SpanContext: ...
+    def with_baggage_item(self: Self, key: str, value: str) -> Self: ...
```

Comment:
This always returns a `SpanContext`, not `self`, so the old annotation looks correct to me.

Reply:
It should depend on the context so current look right